index.html

```diff
@@ -1,12 +1,1449 @@
+                </defs>
+            </svg>
+        </div>
+        <ul class="menu-items">
```

Comment:
menu-items or menu? rethink

Reply:
i have converted the parent name into the navbar

Reply:
done